pkg/util/attributes/parser.go

```diff
@@ -19,29 +19,29 @@ func ReadNodeAttributes(dst *netmap.NodeInfo, attrs []string) error {
 	for i := range attrs {
 		line := replaceEscaping(attrs[i], false) // replaced escaped symbols with non-printable symbols
 
-		words := strings.Split(line, keyValueSeparator)
-		if len(words) != 2 {
+		k, v, found := strings.Cut(line, keyValueSeparator)
+		if !found {
```

Comment:
Maybe `if !found || len(k) == 0 || len(v) == 0` will be better?

Reply:
We already have this check in sdk.